build/package/rpm/go-fdo-server.spec

```diff
@@ -5,7 +5,9 @@
 %global goipath         github.com/fido-device-onboard/go-fdo-server
 %global debug_package   %{nil}
 #%%global tag             v0.0.0
-#%%global commit          08a576eb601fcdb94ee7fa219c16c02834a649e5
+#%%global commit          0
+
+%gometa -L -f
```

Comment:
The current location of `%gometa` is intentional (after the `Version: ` tag), see: https://gitlab.com/fedora/sigs/go/go-rpm-macros/-/blob/main/rpm/macros.d/macros.go-srpm?ref_type=heads#L67

Reply:
NP I can restore that.

I moved it there in order to fix:

`go-fdo-server.src: W: unexpanded-macro URL %{gourl}`

Which also results in the rpm -ql` command spitting out:
`URL         : %{gourl}`

But we can set it manually.